engine/language-server/src/main/scala/org/enso/languageserver/event/InitializedEvent.scala

```diff
@@ -0,0 +1,10 @@
+package org.enso.languageserver.event
+
+/** Event about the initialization of the language server component. */
+sealed trait InitializedEvent extends Event
+
+object InitializedEvent {
+
+  case object SuggestionsRepo  extends InitializedEvent
+  case object FileVersionsRepo extends InitializedEvent
```

Comment:
Please rename it to `SuggestionRepoInitialized` and `FileVersionRepoInitialized`. Your names suggest that it is a repo object.